common/e2e_config/e2e_config.go

```diff
@@ -204,6 +204,7 @@ type E2EConfig struct {
 	ImageTag                     string `yaml:"imageTag" env:"e2e_image_tag"`
 	ImagePullPolicy              string `yaml:"imagePullPolicy" env-default:"IfNotPresent" env:"e2e_image_pull_policy"`
 	InstallLoki                  bool   `yaml:"installLoki" env-default:"true" env:"install_loki"`
+	InstallHaModeEtcd            bool   `yaml:"installHaModeEtcd" env-default:"false" env:"etcd_ha_mode"`
```

Comment:
nit: we name environment variables starting with `e2e_ `, this distinguishes them from other system environment variables.

Reply:
[fixed](https://github.com/openebs/openebs-e2e/compare/96dec989c60afb73f29508c0282b060e198ed707..eff5b4843e198704a234a62f046877ffe5af93f0)